internal/storage/image.go

```diff
@@ -374,6 +372,17 @@ func (svc *imageService) ListImages(systemContext *types.SystemContext, filter s
 	return results, nil
 }
 
+func imageDonePulling(image *storage.Image) bool {
+	donePulling := true
+	for _, name := range image.Names {
+		if _, ok := ImageBeingPulled.Load(name); ok {
+			donePulling = false
```

Comment:
we can just `return false` here, and `return true` below, no need for donePulling if we separate the functions

Reply:
thanks. fixed